uk_bin_collection/uk_bin_collection/councils/CotswoldDistrictCouncil.py

```
@@ -4,240 +4,138 @@
+                        raw_date = (
+                            td.get("data-cell-value", "").strip()
+                            or td.get_text(strip=True)
+                        )
+                        if container_type and raw_date:
                             data["bins"].append({
                                 "type": container_type,
-                                "collectionDate": collection_date
+                                "collectionDate": self._parse_date(raw_date, current_year),
+                            })
+                    except Exception:
+                        continue
```

Ruff (0.15.12)

[error] `try`-`except`-`continue` detected, consider logging the exception

(S112)